docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -6,7 +6,27 @@ description: Describes common issues and solutions for access to servers.
 This section describes common issues that you might encounter in managing access to servers
 with Teleport and how to work around or resolve them.
 
-## Starting SSH sessions fails
+## Starting SSH sessions fails with expired certificates
+
+A ssh session fails to start on certain SSH servers with the error message
+of expired certificates. If a server's date and time is off enough that can
```

Comment:
It's not clear what "off enough" means. Let's be specific.

I think subtract 5 minutes from `time.Now()` when issuing certificates. You'll want to clarify this, but once you do we can then update this section.

Reply:
K, we did see one case that was just 3min causing this.